### DISCUSSION TITLE
customized loss function

### DISCUSSION THREAD

**Discussion by SayyedAhmadKhadem** [MilesCranmer/PySR#270] [Q&A]
Hi,
First of all, I would like to say big thanks to the developers for developing this precious package. It is truly powerful; especially, for using multicores to expedite exploration search .
 
I am going to use your package but my case is not conventional one. I have a particular loss function. Wondering if you could kindly give me ideas/solutions by which the loss function becomes a python code whose inputs are correlations, and then the loss corresponding to each correlation is calculated based on some criteria implemented in the python code.

If you could also share a snippet of code for the solution, that would be an unforgettable help. 

Many thanks for your time and consideration, 
Cheers,
Ahmad

**Comment by MilesCranmer** [comment]
Thanks for the question. See https://github.com/MilesCranmer/PySR/discussions/256 as well as https://github.com/MilesCranmer/PySR/issues/182
